docs/plugin/panel.md

```diff
@@ -51,6 +51,14 @@ spec:
   [ width: <int> ]
 ```
 
+## ScatterChart
+Note: Currently, ther are are no spec options avaiable for this panel
+
+```yaml
+kind: "ScatterChart"
```

Comment:
Very cool PR! Do you see this evolving to support TimeSeries queries and possibly logs in the future? If not, we could potentially lean into this being a plugin that is focused on visualizing traces and rename it to something like TraceViewer (TraceChart sounds a little weird)

Reply:
TBH, I have yet to make plans for TimeSeries and logs support. I wanted to focus on tracing for now. However, it would be a great feature to support multiple data sources in the future.  So, let's try to keep it as 'ScatterChart.'